01-challenge-problem.py

```diff
@@ -1,17 +1,17 @@
-class Point:
+class point:
     def __init__(self, coordX, coordY):
         self.coordX = coordX
```

Comment:
coordX, coordY should be changed, it's redundant naming. Better names are (x, y).